connectors/cassandra/datastore_multi.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I have a serious concerns with checking in commented code. It makes greping the codebase a lot harder and will like be out of dat by the time it get's uncommented. Are you planning on uncommenting this out in future revisions of this PR?